build.gradle

```diff
@@ -26,6 +26,10 @@ intellij {
     pluginName = 'JFrog'
 }
 
+runPluginVerifier {
```

Comment:
Please add a TODO, removing specific verifier version when fixed, include link to the bug